src/pages/guides/how-to-get-funding.md

```diff
@@ -0,0 +1,36 @@
+### How To Get Funding For Your Open Source Projects
+
+# Overview
+
+As wonderful as open source is, one of the biggest challenges maintained projects face is finding funding, especially ones that are sustainable. Now, don’t fret! Here are some suggestions on how to get funding for your open source projects as well as some legalities to consider in determining how to get these resources.
+
+## Direct Sponsorships
+
+Direct sponsorship is the most straightforward way to get funding for your project. When a user or a different company wants to sponsor your project, they send the funds right to your project’s financial account. Here is a list of resources that provide direct sponsorships: \
+
+* **GitHub Sponsors**: This is a popular option that creators use to fund their projects. It allows you to choose the number of tiers you charge and rewards you with potential sponsors. It also provides you with the opportunity to support other developers and organizations, which can be a good way to network with others and gain more support. If you’re interested in using this resource, check out the official website for more information: [https://github.com/sponsors](https://github.com/sponsors)
+* **Open Source Collective**: This is a non-profit organization that is devoted to supporting and promoting open source projects. Like GitHub sponsors, they allow you to choose the number of tiers you charge and reward you with potential sponsors. In contrast, Open Source Collective provides information on how the money is being used. If interested, check out their website for more information: [https://opencollective.com/how-it-works](https://opencollective.com/how-it-works)
```

Comment:
[Proofreading] A few issues here:

* There's no "Source" in their name..
* They're a company, not a non-profit.
* I find the phrasing _"reward(s) you with"_ to be a bit confusing. Is it that you may potentially get the reward of sponsors joining? I'd think something like _"allow"_ or _"enable"_ might be a better word here than _"reward"_.
* I think this should also mention [StackAid](https://www.stackaid.us), [Thanks.dev](https://thanks.dev), and [Tidelift](https://tidelift.com/lifter)

Reply:
Thanks for the feedback @JoshuaKGoldberg! I did mention the sponsors you listed. I just added them in the aggregated section.